### PR TITLE
feat. edit panel for motd

### PR DESCRIPTION
**Pull request by nivcoo**

### CODE REVIEW

File: app/Controller/AppController.php [hunk 1]
```diff
@@ -242,6 +242,11 @@ public function __initAdminNavbar()
                         'icon' => 'bars',
                         'permission' => 'MANAGE_NAV',
                         'route' => ['controller' => 'navbar', 'action' => 'index', 'admin' => true, 'plugin' => false]
+                    ],
+                    'Motd' => [
```
Comment (Eywek) [inline app/Controller/AppController.php]: Use `MOTD__TITLE`

File: app/Controller/MotdController.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        $get_all_servers = $this->Server->findSelectableServers(false);
+        $get_servers = [];
+        $calls = ['GET_MOTD' => []];
+
+        foreach ($get_all_servers as $key => $value) {
+            if (!$this->ServerComponent->online($key))
+                continue;
+            $call = array_values($this->ServerComponent->call($calls, $key));
+            $this->log($call);
```
Comment (Eywek) [inline app/Controller/MotdController.php]: Why this log?

File: app/Controller/AppController.php (hunk 1 above)
Comment (nivcoo) [reply app/Controller/AppController.php]: Okey

File: app/Controller/MotdController.php (hunk 2 above)
Comment (nivcoo) [reply app/Controller/MotdController.php]: I forgot it during a debug

**Comment by Eywek** [comment]
`This branch is out-of-date with the base branch`
Update ta branche par rapport a dev

**Comment by Eywek** [comment]
👍

**Comment by MaximeMichaud** [comment]
Faudra ajouter : https://github.com/nivcoo/MineWebCMS/pull/1/commits/8f2f69c8244974c26f0607781b561fb30dec06bf

**Comment by nivcoo** [comment]
C'est bon on peut merge

**Comment by MaximeMichaud** [comment]
👍

**Comment by DarkScientist** [comment]
👍

**Comment by Eywek** [comment]
:+1:

**Pull request merged**